docker/compose.yaml

```diff
@@ -28,6 +29,33 @@ services:
     ports:
       - 127.0.0.1:${ES_8_PORT:-9208}:9200
   
+  elasticsearchFollower:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.13.2
+    container_name: es-follow
+    profiles: ["ccr"]
+    environment:
+      - cluster.name=es-follow
+      - bootstrap.memory_lock=true
+      - discovery.type=single-node
+      - xpack.security.enabled=false
+      - xpack.watcher.enabled=false
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
Can we upgrade this to 1G for XMX & XMS? I've seen flakyness before with this little memory assigned to the JVM.